applications/plugins/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.12)
 find_package(SofaFramework)
 
 sofa_add_subdirectory_external(SofaHighOrder SofaHighOrder)
-sofa_add_subdirectory_external(SofaPython3 SofaPython3)
+#sofa_add_subdirectory_external(SofaPython3 SofaPython3)
```

Comment:
???

Reply:
Collateral damage 😅 (will fix it along onther things) 
but I could not just only disable Python3 compilation in the cmake by the way